packages/client/src/index.ts

```diff
@@ -134,7 +152,19 @@ export default class LogtoClient {
    * resource, as specified in the Logto Console.
    */
   async getAccessTokenClaims(resource?: string): Promise<AccessTokenClaims> {
-    const accessToken = await this.getAccessToken(resource);
+    // eslint-disable-next-line unicorn/no-useless-undefined
+    const accessToken = await this.getAccessToken(resource, undefined);
```

Comment:
Why the explicit `undefined`?

Reply:
was trying to use `undefined` for more clear function calling, but removed now.